include/openssl/dh.h

```diff
@@ -29,6 +29,9 @@ extern "C" {
 # ifndef OPENSSL_DH_MAX_MODULUS_BITS
 #  define OPENSSL_DH_MAX_MODULUS_BITS    10000
 # endif
+# ifndef OPENSSL_DH_MAX_CHECK_MODULUS_BITS
+#  define OPENSSL_DH_CHECK_MAX_MODULUS_BITS  32768
```

Comment:
Should it be `OPENSSL_DH_MAX_CHECK_MODULUS_BITS` or `OPENSSL_DH_CHECK_MAX_MODULUS_BITS`?

Reply:
Fixed